src/utils/vulnerability.ts

```diff
@@ -76,7 +76,16 @@ export function processAuditJson(
     return Object.values(advisories).reduce(
       (acc: ProcessedResult, cur: v6Advisory) => {
         const shouldAudit = mapLevelToNumber(cur.severity) >= mapLevelToNumber(auditLevel);
-        const isExcepted = exceptionIds.includes(Number(cur.id));
+        let isExcepted: boolean = false;
+
+        if (cur.id && exceptionIds.includes(Number(cur.id)) || // NPM v6 contains 'id's to use
+            (cur.cves && exceptionIds.filter(id => cur.cves.includes(id)).length > 0) || // NPM v6 can also have an array of cve id's
+            (cur.via && cur.via[0].source && exceptionIds.includes(Number(cur.via[0].source))) || //auditReportVersion: 2. Check via.source for id
+            (cur.via && cur.via[0].url && exceptionIds.filter(id => cur.via[0].url.contains(id)).length > 0 )) //auditReportVersion: 2. Check via.url for github id
```

Comment:
this seems to be v7/v8 report structure, might need to move this checking to below [v7 handling](https://github.com/jeemok/better-npm-audit/pull/73/files#diff-ea566fb8b530d96f3dfa855f2c5aea4b33edfe5f8f4cb8329b92d64cdba3edeaR130)

Reply:
Hey @jeemok. Yup I would say the v7/8 work should be down with the v7 handling, sorry about that! Glad you're able to take a look, I was really just hoping to get some ideas out there on how we could improve the ID stability a bit.